components/apimgt/org.wso2.carbon.apimgt.common.jms/src/main/java/org/wso2/carbon/apimgt/common/jms/JMSTransportHandler.java

```diff
@@ -50,7 +51,17 @@ public class JMSTransportHandler {
     private boolean stopIssued = false;
     private static final Object lock = new Object();
 
-    public JMSTransportHandler(Properties jmsConnectionProperties) {
+    /**
+     * Constructs a JMSTransportHandler with specified JMS connection properties and JMS task manager properties.
+     *
+     * @param jmsConnectionProperties  The JMS connection properties to utilize.
+     *                                 If empty, default properties will be loaded.
+     * @param jmsTaskManagerProperties The properties for JMS task management.
+     *                                 If null, default values will be applied.
```

Comment:
We only have one param here right? We can remove the jmsConnectionProperties from the doc comment.